### PR TITLE
Add exceptions hierarchy

### PR DESCRIPTION
**Pull request by coungard**
Javadocs in process

### CODE REVIEW

**Comment by coungard** [comment]
> 
> 
> Javadocs in process

File: src/main/java/com/reckue/post/exceptions/ReckueException.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.reckue.post.exceptions;
+
+/**
+ * Class ReckueException it is a base abstract class for all other exceptions.
```
Comment (camelya58) [inline src/main/java/com/reckue/post/exceptions/ReckueException.java:4]: Delete unnecessary "it".

File: src/main/java/com/reckue/post/exceptions/ReckueIllegalArgumentException.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.reckue.post.exceptions;
+
+import lombok.Getter;
+
```
Comment (camelya58) [inline src/main/java/com/reckue/post/exceptions/ReckueIllegalArgumentException.java:4]: Please, add java docs.

Comment (coungard) [reply src/main/java/com/reckue/post/exceptions/ReckueIllegalArgumentException.java:4]: i hate this step)

**Pull request closed without merging**